src/screens/Dashboard/DashboardScreen.tsx

```
@@ -80,7 +80,43 @@ function DashboardScreen(): JSX.Element {
+    const userAgent = navigator.userAgent || navigator.vendor;
+
+    if (/iPad|iPhone|iPod/.test(userAgent)) {
+      return 'iOS';
+    }
+
+    if (/android/i.test(userAgent)) {
+      return 'Android';
+    }
+  }
   React.useEffect(() => {
+    if (false) {
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)